### PR TITLE
Add github templates for bug, rfe, PRs

### LINKED ISSUE #757: Create a github template for bug/rfe etc

### PR DESCRIPTION
**Pull request by sasagarw**
This PR:
- Adds github template for bugs report.
- Adds github template for new feature request.
- Adds github template for pull request.

Closes: #757

### CODE REVIEW

**Comment by leelavg** [comment]
Finally!

Hey @sasagarw, thanks for taking this up! will review at the earliest.

**Review by amarts — COMMENTED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+3. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what actually happened.
+
+**Environment:**
+ - Kadalu Version: [e.g., 0.8.6]
```
Comment (leelavg) [inline .github/ISSUE_TEMPLATE/bug_report.md:44]: - a comment about getting the version here would be helpful, something along the lines
```
<!--
Output from one of commands
1. If kubectl-kadalu plugin is installed
$ kubectl kadalu version

2. If plugin isn't installed
$ kubectl exec -it deploy/operator -n kadalu -- bash -c 'kubectl-kadalu version'
-->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ - K8S_DIST: [e.g., kubernetes, openshift]
```
Comment (leelavg) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: - just a memo, we better add support for getting the k8s_dist as well from version command.

**Review by leelavg — COMMENTED** [review]
- uh oh! missed review this 😅 
- overall these are good addition, thanks again.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (sasagarw) [reply .github/ISSUE_TEMPLATE/bug_report.md:45]: @leelavg `k8s_dist` is just an environment variable right?

Comment (sasagarw) [reply .github/ISSUE_TEMPLATE/bug_report.md:45]: Or are you expecting a command to read the yaml file from cluster and from there check this env var value?

Comment (leelavg) [reply .github/ISSUE_TEMPLATE/bug_report.md:45]: Yes, that's the idea however in a running cluster I don't think we can find that directly as the value is interpolated by that time iirc.

Anyways, it's just a todo not related to this PR.

**Comment by sasagarw** [comment]
@leelavg can we merge it now then?

**Review by leelavg — APPROVED** [review]
lgtm

**Pull request merged**